rosmon_core/src/main.cpp

```diff
@@ -75,6 +75,9 @@ void usage()
 		"                 Use GROUP as name of the launch group. By default, empty\n"
 		"  --launch-config=CONFIG\n"
 		"                 Use CONFIG as name of the launch config. By default, empty\n"
+                "  --respawn-attr=force_true|force_false\n"
```

Comment:
please check formatting in the file, replace space indentation with tabs. The simplest way to make sure is to view spaces in IDE (https://stackoverflow.com/questions/30140595/show-whitespace-characters-in-visual-studio-code) and replace spaces with tabs (copy from another part part of file)